src/ServerTelemetryChannel/SamplingTelemetryProcessor.cs

```diff
@@ -45,6 +44,11 @@ public SamplingTelemetryProcessor(ITelemetryProcessor next)
 
             this.SamplingPercentage = 100.0;
             this.Next = next;
+            if (this.UnsampledNext == null)
```

Comment:
I think it might be always true

Reply:
Yes, I've removed the if statement and am just setting Next = UnsampledNext in the public constructor.